frontend/src/pages/chartTest.js

```diff
@@ -1,107 +1,157 @@
-    const [testInput1, settestInput1] = useState(0);
-    const [testInput2, settestInput2] = useState(0);
-    const [testInput3, settestInput3] = useState(0);
-
-
-    const scenario_scoring_matrix = {
-        Tim: {
-            issue1: 1,
-            issue2: 4,
-            issue3: 2
-        },
-        Song: {
-            issue1: 0,
-            issue2: 2,
-            issue3: 3
-        }
-
-    };
-
+  const [input, setInput] = useState(0);
+  const [testInput1, setTestInput1] = useState(0);
+  const [testInput2, setTestInput2] = useState(0);
+  const [testInput3, setTestInput3] = useState(0);
+
+  const scenario_scoring_matrix = {
+    Tim: {
+      issue1: 1,
+      issue2: 4,
+      issue3: 2,
+    },
+    Song: {
+      issue1: 0,
+      issue2: 2,
+      issue3: 3,
+    },
+  };
+  function Total_EmphasisF(scenario_scoring_matrix) {
     const Total_Emphasis = {};
     for (let conversant in scenario_scoring_matrix) {
-        for (let issue in scenario_scoring_matrix[conversant]) {
-            Total_Emphasis[issue] = (Total_Emphasis[issue] === undefined) ? scenario_scoring_matrix[conversant][issue] : scenario_scoring_matrix[conversant][issue] + Total_Emphasis[issue]
-        }
+      for (let issue in scenario_scoring_matrix[conversant]) {
+        Total_Emphasis[issue] =
+          Total_Emphasis[issue] === undefined
+            ? scenario_scoring_matrix[conversant][issue]
+            : scenario_scoring_matrix[conversant][issue] +
+              Total_Emphasis[issue];
+      }
+      return Total_Emphasis;
     }
-    // a perfect student
+  }
+  const Total_Emphasis = Total_EmphasisF(scenario_scoring_matrix);
+  // a perfect student
+
+  function Student_EmphasisF(scenario_scoring_matrix) {
     let Student_Emphasis = {};
     if (testInput1 === 0 && testInput2 === 0 && testInput3 === 0) {
-        for (let conversant in scenario_scoring_matrix) {
-            for (let issue in scenario_scoring_matrix[conversant]) {
-                Student_Emphasis[issue] = (Student_Emphasis[issue] === undefined) ? scenario_scoring_matrix[conversant][issue] : scenario_scoring_matrix[conversant][issue] + Student_Emphasis[issue]
-            }
+      for (let conversant in scenario_scoring_matrix) {
+        for (let issue in scenario_scoring_matrix[conversant]) {
+          Student_Emphasis[issue] =
+            Student_Emphasis[issue] === undefined
+              ? scenario_scoring_matrix[conversant][issue]
+              : scenario_scoring_matrix[conversant][issue] +
+                Student_Emphasis[issue];
         }
+      }
     } else {
-
-        Student_Emphasis = {
-            issue1: testInput1,
-            issue2: testInput2,
-            issue3: testInput3,
-        }
+      Student_Emphasis = {
+        issue1: testInput1,
+        issue2: testInput2,
+        issue3: testInput3,
+      };
     }
+    return Student_Emphasis;
+  }
+  let Student_Emphasis = Student_EmphasisF(scenario_scoring_matrix);
```

Comment:
same here, normally variables don't start upper cased, that's generally reserved for components. Not a huge deal though if this is what you prefer